src/main/java/core/basesyntax/service/parse/DataParserImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.parse;
+
+import core.basesyntax.model.FruitRecord;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of a class or method implementation. Please remove the empty line 8.